src/main/java/frc/robot/RobotContainer.java

```diff
@@ -54,11 +54,16 @@ public class RobotContainer {
   // driving in open loop
   private final SwerveRequest.SwerveDriveBrake brake = new SwerveRequest.SwerveDriveBrake();
   private final SwerveRequest.PointWheelsAt point = new SwerveRequest.PointWheelsAt();
-  private final Telemetry logger = new Telemetry(MaxSpeed);
+  private final Telemetry logger;
+
+  private final DynamicRobotConfig dynamicRobotConfig = DynamicRobotConfig.loadDynamicRobotConfig();
```

Comment:
Any reason not to initialize this within the constructor, like you are doing with the other variables you've changed in this class?